src/components/Main/MainView/MessageInput/MessageInputFileList.vue

```diff
@@ -37,7 +37,13 @@ export default defineComponent({
 .container {
   width: 100%;
   display: flex;
-  padding: 16px;
+  padding: {
+    left: 16px;
+    right: 16px;
+    bottom: 12px;
+  }
```

Comment:
```suggestion
  padding: 4px 0;
  margin-bottom: 8px;
```
スクロールバーの高さが8pxなので4になります
`margin-top`じゃなくて`padding-top`でいい気がします

スクロールバーの下に余白がほしいです(`margin-bottom`)
内側から padding / scrollbar / border / marginの順になってます